flytekit/types/schema/types.py

```diff
@@ -236,25 +243,13 @@ def __init__(
 
         if local_path is None:
             local_path = FlyteContextManager.current_context().file_access.get_random_local_directory()
-        self._local_path = local_path
-        self._remote_path = remote_path
-        self._supported_mode = supported_mode
+        self.local_path = local_path
```

Comment:
is there a reason we needed to remove the underscores?  not a big deal but this is the pattern we use in other parts of the codebase.  worth it to add a comment if we need to digress from it

Reply:
I've updated it